src/components/atoms/InfoBox/infoBox.atom.types.ts

```diff
@@ -0,0 +1,6 @@
+export interface IInfoBox {
+  usdValue?: string;
```

Comment:
Why not treat this as a generic header? That way it's not strictly binded to a specific domain value but can be used more generically